src/callback.rs

```diff
@@ -58,17 +56,17 @@ where
     }
 }
 
-pub(crate) unsafe fn register_call_result<C, F, Manager>(
-    inner: &Arc<Inner<Manager>>,
-    api_call: sys::SteamAPICall_t,
-    _callback_id: i32,
-    f: F,
-) where
-    F: for<'a> FnOnce(&'a C, bool) + 'static + Send,
-{
-    let mut callbacks = inner.callbacks.lock().unwrap();
-    callbacks.call_results.insert(
-        api_call,
-        Box::new(move |param, failed| f(&*(param as *const C), failed)),
-    );
-}
+// pub(crate) unsafe fn register_call_result<C, F, Manager>(
+//     inner: &Arc<Inner<Manager>>,
+//     api_call: sys::SteamAPICall_t,
```

Comment:
Is this maybe still there? Might not want to comment it out if we use the file and contents compile successfully.